packages/application-shell-connectors/src/components/application-context/index.js

```diff
@@ -1,4 +1,5 @@
 export {
+  Context,
```

Comment:
The React docs state a useContext call needs the return of `createContext`. Not only the consumer.